lambda_function.py

```
@@ -3,223 +3,290 @@
-                    print(f"Warning: Invalid coordinates generated for point: x={x}, y={y}, z={z}")
-            except Exception as e:
-                print(f"Error processing point ({x},{y},{z}): {e}")
-
-        # Upload the georeferenced trajectory to S3
-        s3.put_object(
-            Bucket=bucket,
-            Key=output_key,
-            Body=output.getvalue()
-        )
+                    print(f"Warning: invalid coordinates for {trajectory['photo_id']}")
+            except Exception as exc:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)